docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -121,77 +120,77 @@ The NEB part of CASTEP should pick up on the imposed ionic constraint(s) and you
  
 in the `.castep` output file.
 
+*[Shouldn't this be referenced in the main documentation, as it looks like a specific feature of the NEB code?]*
 
 ### Do I need climbing NEB?
 
-The climbing NEB method modifies the force on the highest-energy image (usually close to the middle of the band) such that it tries to climb *up* the barrier. 
-This is very useful to get accurate barrier estimates, but care must be taken when more complex pathways (i.e. with multiple maxima) are found.
+The climbing NEB method modifies the force on the highest-energy image (usually close to the middle of the band) *[? What's a band?]* such that it tries to climb *up* the barrier. 
+This is very useful to get accurate barrier estimates, but care must be taken when more complex pathways (i.e. with multiple maxima) are found.  *[Don't get this!]*
```

Comment:
I could be wrong, but if you have multiple TS states, the CI NEB will still only make one of the images climb up, so you might get a distorted path as a result. In such cases it's probably best to break the path into multiple single-TS NEB calculations, fully relaxing any meta-stable intermediate structures.

Reply:
OK, should have connected "band" to NEB, but it's not clear to me what this band *is*. On the one hand, I visualised the band as the forces that help find the solution, on the other is it the solution itself - the final energy profile that "hugs" the minimum energy path.

Intuitively it does seem reasonable that a complex profile might need to be split into individual steps between metastable transition states.

Presumably things also get tricky if there are multiple pathways? You can presumably have a global vs. local minimisation problem? 

I think this is getting beyond the scope of the tutorial. I would emphasise that the tutorial is geared towards a simple case where you are not expecting metastable intermediates (a separate tutorial if demand?).

I would clarify what is meant by "climbing NEB method (or mode?)" vs. it's opposite.

Reply:
I agree that it goes beyond the scope of the tutorial. I'll add a TODO item below to clarify this part of the tutorial and include advice about starting with regular NEB and turning on climbing image after a few steps.